src/components/common/Pagination.jsx

```diff
@@ -1,28 +1,35 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
 import {Button, Grid} from './'
 
-const Pagination = () => {
+import {PAGINATION_TYPE} from '../../utils/constants'
+
+const Pagination = ({activePagination, totalPages, currentPage}) => {
```

Comment:
There is an scenario where I think you are missing Pagination, look at 5.3.2 on Capstone Project Repo. Type 'a' on your search input, and look that the request is returning more than 20 results, buy you are only showing the first page and no pagination controls are shown :), you still have time to fix it!